docs/content/guidelines/usage.mdx

```diff
@@ -5,74 +5,93 @@ title: Usage guidelines
+| <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="16" height="16"><path fill-rule="evenodd" d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0016 8c0-4.42-3.58-8-8-8z"></path></svg> | **GH mark** | Brand logo use only. Follow [GitHub's brand guidelines](https://brand.github.com/) when using this octicon.
 
-### Octicons custom to `git`
+## Icon styling
 
-The following should only be used for their specific meanings:
+By default, all Octicons use an outline style. Some icons have a filled version which should be use only under the following conditions:
 
-- Commits: The commit icon is represented as a circle on a line. This is to symbolize a point (the circle) along the version history timeline in which changes were made. This icon should be used in views to represent individual or groups of commits. These views include the timeline and the Commits tab in pull requests.
-- Issues: The issue icon is represented as a circle with an `!` mark. The base issue icon is extended through slight modifications to communicate different states of an issue: a checkmark for a closed issue and combining the sync icon with an `!` for reopening an issue. These three symbols should be used exclusively with issues.
-- Pull requests: The pull request icon is represented as a commit dot attached with an arrow being inserted into the history timeline (two dots connected by a line).
-- Repos: A repository in GitHub is represented as a closed book and is explicitly named as `repo`. Octicons does contain a `book` icon which displays an open book symbol that is used for wikis, READMEs, and other use cases where the user can see a collection of information.
+<Do>
+    <img width="456" alt="octicons-style-pairing" src="https://user-images.githubusercontent.com/10384315/103719162-490c9600-4f7d-11eb-8d9c-1d867ee51de9.png" />
+    <Caption>To aid in contrast when in relation to another icon</Caption>
+</Do>
 
```

Comment:
I noticed there's no margin between these 2 do components, at least in [this version](https://octicons-e76mon3x5.vercel.app/octicons/guidelines/usage). Is that something we need to add manually?